tekton/ci/jobs/tekton-integration-tests.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
FYI - this exists too https://github.com/tektoncd/plumbing/blob/main/tekton/ci/jobs/e2e-kind.yaml

Reply:
Thanks. I'll update the template to use e2e-kind. Unless that exists too ;)